docs/contribute/cumulative-docs/example-scenarios.md

```diff
@@ -0,0 +1,545 @@
+```
+::::
+:::::
+::::::
+
+### If the section lifecycle is different than the page level [unversioned-added-different]
+
+For example, on the [Dashboard controls](https://www.elastic.co/docs/explore-analyze/dashboards/add-controls#add-esql-control) page we added content about new ES|QL controls functionality that was added to Serverless in preview.
+Since this is different than the page-level applicability in the frontmatter, `serverless: ga`,
+label the content about the new functionality with `serverless: preview`.
+
+:::::{tab-set}
+::::{tab-item} Image
+:::{image} ./images/example-unversioned-added-different.png
+:screenshot:
+:alt:
+:::
+::::
+::::{tab-item} Code
+````
+---
+serverless: ga
+---
+
+# Add filter controls
+
+[...]
+
+## Add ES|QL controls
+
+```{applies_to}
+serverless: preview
+```
+
+[...]
+````
+::::
+:::::
+
+## Functionality changes lifecycle state [lifecycle-changed]
+
+When the functionality described in any content changes lifecycle state,
+how it is labeled varies by whether the product/deployment mode is versioned or unversioned.
+
+For example, the majority of the content on the [Lens](https://www.elastic.co/docs/explore-analyze/visualize/lens)
+page was applicable to both Elastic Stack and to Serverless.
+One specific section describes functionality that was in technical preview in Elastic Stack 9.0.0 and
+Serverless at the time Elastic Stack 9.0.0 was released.
+Then, the functionality became generally available in Elastic Stack in 9.1.0 and shortly before the
+Elastic Stack 9.1.0 release in Serverless.
+
+* For Elastic Stack, a versioned product, label the section with both lifecycles: `ga 9.1` and `preview 9.0`.
+* For Serverless, an unversioned product, update the section label from `serverless: preview` to `serverless: ga`.
+  Do _not_ list both lifecycles.
+
+:::::{tab-set}
+::::{tab-item} Image
+:::{image} ./images/example-lifecycle-changed.png
+:screenshot:
+:alt:
+:::
+::::
+::::{tab-item} Code
+````
+---
+applies_to:
+  stack: ga
+  serverless: ga
+---
+
+# Lens [lens]
+
+[...]
+
+### Assign colors to terms [assign-colors-to-terms]
+
+```{applies_to}
+stack: ga 9.1, preview 9.0,
+serverless: ga
+```
+
+[...]
+
+````
+::::
+:::::
+
+
+## Functionality is removed [removed]
+
+When the functionality described in any level of content is removed,
+how to handle it varies by whether the product/deployment mode is versioned or unversioned.
+
+For example, we removed the `securitySolution:enableVisualizationsInFlyout` setting that was described on the
+[Configure advanced settings](https://www.elastic.co/docs/solutions/security/get-started/configure-advanced-settings)
+page from the Elastic Stack in 9.1.0 and from Serverless around the same time.
+Since this this functionality is still available before 9.1.0, we need that content to continue to be
+available to users on Elastic Stack earlier versions while communicating to users on newer versions
+that it is no longer available.
```

Comment:
could link down to exceptions for tech preview/beta functionality here

Reply:
I rearranged some things in https://github.com/elastic/docs-builder/pull/1614/commits/a2c5e6ecede20691791acbde7cd2e66aa72be455 to group all the information about tagging content for removed functionality in one place: [preview](https://docs-v3-preview.elastic.dev/elastic/docs-builder/pull/1614/contribute/cumulative-docs/example-scenarios#removed).